kinetic/distribution.yaml

```diff
@@ -1317,6 +1317,21 @@ repositories:
       url: https://github.com/ros/cmake_modules.git
       version: 0.4-devel
     status: maintained
+  cnn_bridge:
+    doc:
+      type: git
+      url: https://github.com/wew84/cnn_bridge.git
+      version: master
+    release:
+      tags:
+        release: release/kinetic/{package}/{version}
+      url: https://github.com/wew84/cnn_bridge.git
```

Comment:
This is not a release repository. Did you intend to make a binary release? If so please follow: http://wiki.ros.org/bloom/Tutorials/FirstTimeRelease